types/meteor/webapp.d.ts

```diff
@@ -28,6 +28,16 @@ declare module 'meteor/webapp' {
         var connectApp: connect.Server;
         function suppressConnectErrors(): void;
         function onListening(callback: Function): void;
+
+        type RuntimeConfigHookCallback = (options: {
+            arch: 'web.browser' | 'web.browser.legacy' | 'web.cordova';
+            request: http.IncomingMessage;
+            encodedCurrentConfig: string;
+            updated: boolean;
+        }) => string | undefined | null | false;
+        function addRuntimeConfigHook(callback: RuntimeConfigHookCallback): void;
+        function decodeRuntimeConfig(rtimeConfigString: string): any;
+        function encodeRuntimeConfig(rtimeConfig: any): string;
```

Comment:
We should use `unknown` instead of `any`.

Reply:
I'm not sure how I would use that as a consumer. Since the recommended pattern is something like

```js
const config = WebApp.decodeRuntimeConfig(encodedCurrentConfig);
config.newVar = 'some value';
config.oldVar = 'new value';
return WebApp.encodeRuntimeConfig(config);
```

if `config` is `unknown`, what am I supposed to do with it to be able to assign values? It seems like any user of the function would be forced to cast to `any` regardless.

Reply:
Presumably the goal is to force casting, not to `any` but the object type that actually gets used?
```js
interface MyConfig {
  newVar: string;
  oldVar: string;
};
const config = WebApp.decodeRuntimeConfig(encodedCurrentConfig) as MyConfig;
config.newVar = 'some value';
config.oldVar = 'new value';
return WebApp.encodeRuntimeConfig(config);
```
If you use `any`, the user won't know that there's any typing work to be done here.

Reply:
Hmm. Alright, I don't know that that's the choice I would have made, but I can get behind it. Pushed a fix.

Reply:
I’m glad you both solved it!